Jenkinsfile

```diff
@@ -26,9 +26,25 @@ elifePipeline {
         }
 
         elifeMainlineOnly {
-            stage 'Push image', {
-                image.push()                    
-                image.tag('latest').push()
+            stage 'Merge to master', {
+                elifeGitMoveToBranch commit, 'master'
+            }
+
+            stage 'Push unstable image', {
+                def unstable_image = image.addSuffixAndTag('_unstable', commit)
+                unstable_image.push()
+                unstable_image.tag('latest').push()
+            }
+        }
+
+        elifeTagOnly { tagName ->
+            def releaseTag = tagName - "v"
+
+            elifeMainlineOnly {
```

Comment:
what is point in having this `elifeMainlineOnly` block?

Reply:
I guess there is no point and I had just messed it up. Will remove it.